src/components/PDFView/PDFPasswordForm.js

```diff
@@ -0,0 +1,164 @@
+import _ from 'underscore';
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {View, ScrollView} from 'react-native';
+import Button from '../Button';
+import Text from '../Text';
+import TextInput from '../TextInput';
+import Icon from '../Icon';
+import * as Expensicons from '../Icon/Expensicons';
+import styles from '../../styles/styles';
+import colors from '../../styles/colors';
+import PDFInfoMessage from './PDFInfoMessage';
+import compose from '../../libs/compose';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../withWindowDimensions';
+
+const propTypes = {
+    /** If the submitted password is invalid (show an error message) */
+    isPasswordInvalid: PropTypes.bool,
+
+    /** If the password field should be auto-focused */
+    shouldAutofocusPasswordField: PropTypes.bool,
+
+    /** If loading indicator should be shown */
+    shouldShowLoadingIndicator: PropTypes.bool,
+
+    /** Notify parent that the password form has been submitted */
+    onSubmit: PropTypes.func,
+
+    /** Notify parent that the password has been updated/edited */
+    onPasswordUpdated: PropTypes.func,
+
+    /** Notify parent that a text field has been focused or blurred */
+    onPasswordFieldFocused: PropTypes.func,
+
+    ...withLocalizePropTypes,
+    ...windowDimensionsPropTypes,
+};
+
+const defaultProps = {
+    isPasswordInvalid: false,
+    shouldAutofocusPasswordField: false,
+    shouldShowLoadingIndicator: false,
+    onSubmit: () => {},
+    onPasswordUpdated: () => {},
+    onPasswordFieldFocused: () => {},
+};
+
+class PDFPasswordForm extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            password: '',
+            validationErrorText: '',
+            shouldShowForm: false,
+        };
+        this.submitPassword = this.submitPassword.bind(this);
+        this.validate = this.validate.bind(this);
```

Comment:
This method is only called from `submitPassword()` and `validateAndNotifyPasswordBlur()` which are already bound to `this`, so this line can be removed.